programs/pkey/pk_sign.c

```diff
@@ -20,6 +20,7 @@
 #include "mbedtls/build_info.h"
 
 #include "mbedtls/platform.h"
+#include "mbedtls/md.h"
```

Comment:
Note: we already include this file a few lines below. Perhaps remove the 2nd one? Otherwise in some time, someone might notice the duplication and be tempted to remove this one.

(Btw, should we add a comment along the lines of "include this early because we need MD_CAN macros"?)

(Same for all sign/verify programs in this directory.)